### ISSUE TITLE
rename filename to title is not supporting frontmatter title stype

### ISSUE THREAD

**Issue by ariccb** [NotePlan/plugins#590]
**Checks**
- [ ] I confirm I have restarted NotePlan and this problem still persists.
- [ ] Is this a repeatable problem?

**Describe the bug**

What do you expect it to do instead?

**To Reproduce**
have a file with frontmatter title like 
```
---
title: test title
---
```
**Screenshots**
![Screenshot_2024-08-15_NotePlan Beta_034203PM](https://github.com/user-attachments/assets/5cdacae8-e7c9-41fb-aef6-2343ef9f9abf)

**System**
 - Device: mac
 - OS: Sonoma 14.5 (23F79)
 - NotePlan Version [e.g. 3.0.25]
 - Plugin Name & Version Note Helpers v0.19.2

**Additional context**

**Plugin Console Log**
To provide us with more clues about where the bug/error is occurring...
1. Open the Plugin's Preferences by going to `NotePlan's menu > Preferences > Plugins` and clicking the settings "cog" icon next to the plugin in question. Scroll to the bottom and set the logging level to "DEBUG" and click "Save & Close"
3. Now open the Plugin Console by going to `Noteplan > Help > Plugin Console` (_not_ the macOS Console app).
4. Run the plugin command you're reporting
5. Copy the output from the Plugin Console and paste it below
6. Delete any output that has personal information you don't want in there
```javascript
    PASTE_PLUGIN_CONSOLE_OUTPUT_HERE
```

**Comment by jgclark** [comment]
Thanks for the report. I've added it to my list.

**Comment by jgclark** [comment]
Will be released in v0.20.
